website/content/blog/Towards-Optimized-Microservices-Performance-Sustainability-via-Istio-Kepler-and-Smart-Scheduling.md

```diff
@@ -0,0 +1,64 @@
+# Towards Optimized Microservices Performance &  Sustainability via Istio, Kepler and Smart Scheduling
```

Comment:
```suggestion
---
 title: Towards Optimized Microservices Performance & Sustainability via Istio, Kepler and Smart Scheduling
 description: Optimized Microservices Performance & Sustainability via Istio, Kepler and Smart Scheduling
 slug: 2023-sustainability-istio-kepler-smart-scheduling
 weight: 1
 date:   2023-10-12 00:00:00 +0000
 author: Peng Hui Jiang
 ---
 
 *Guest post as part of the [cloud native sustainability week](https://tag-env-sustainability.cncf.io/cloud-native-sustainability-week/)*
```

Reply:
@jiangphcn feel free to update the description with whatever you feel works best here, thanks!